### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by RoXuS**
Hello !

I have added a way to set the submit text (to handling i18n).
I have set the loading to false in case of error on form.

Thx !

### CODE REVIEW

File: stripe-card.html [hunk 1]
```diff
@@ -74,7 +74,7 @@
         </div>
         <div id="error"></div>
         <template is="dom-if" if="[[ !hideSubmit ]]">
-          <paper-button on-click="_submit">Submit</paper-button>
+          <paper-button on-click="_submit">[[submitLabel]]</paper-button>
```
Comment (morbidick) [inline stripe-card.html:77]: Looks good and makes sense

File: stripe-card.html [hunk 2]
```diff
@@ -238,7 +242,9 @@
       }
 
       _submit() {
-        this.createToken().catch((error) => {/* suppress uncaught promise warnings */});
+        this.createToken().catch((error) => {
+          this.loading = false;
```
Comment (morbidick) [inline stripe-card.html]: This should be already done by the catch in the `createToken` function

**Review by morbidick — CHANGES_REQUESTED** [review]
Thanks for the label improvement, would you mind removing the second commit? that case is already handled. I'll look into updating the failing CI.

**Comment by RoXuS** [comment]
Done, thx for your response !

**Comment by RoXuS** [comment]
@morbidick I have tested without the `this.loading = false` in promise catch and it is not handled by the try/catch, so after an error on form, the loading is never set to false !

**Comment by morbidick** [comment]
CI updated and version 2.5.0 released :tada:

**Pull request merged**